process-migration-service/frontend/package-lock.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@babel/cli": {
       "version": "7.5.5",
-      "resolved": "https://repository.engineering.redhat.com/nexus/repository/registry.npmjs.org/@babel/cli/-/cli-7.5.5.tgz",
+      "resolved": "https://registry.npmjs.org/@babel/cli/-/cli-7.5.5.tgz",
```

Comment:
change not required. remove this file from the commit

Reply:
removed